assets/src/js/frontend/paypal-commerce/AdvancedCardFields.js

```diff
@@ -55,7 +55,7 @@ class AdvancedCardFields extends PaymentMethod {
 	static canShow() {
 		return paypal?.HostedFields?.isEligible() === true
             && '1' === window.givePayPalCommerce.supportsCustomPayments
-            && 'auto' === givePayPalCommerce.paymentFieldType;
+            && givePayPalCommerce.payPalSdkQueryParameters.components.indexOf('hosted-fields') !== -1;
```

Comment:
These `indexOf` checks could be simplified using `includes()`.

Reply:
@kjohnson I am doing this because `components` is a string.